tox.ini

```diff
@@ -43,10 +43,11 @@ deps =
     flake8
     flake8-black
     flake8-isort
+    flake8-use-fstring
```

Comment:
@pyrco thoughts on adding this?

Reply:
I think we shouldn't add this. We keep the number of flake8 extensions limited on purpose, as they often have unwanted side-effects. This one included as it needs `--enable-extensions=FS003` to catch this type of bug, which may trigger on legitimate `{}` in strings that er not f-strings or are templates.

Reply:
Alrighty, I'll get rid of these changes then. 👍